xdsl/__init__.py

```diff
@@ -1,3 +1,79 @@
+from collections.abc import Sequence
+from itertools import chain
+from typing import Any
+
+from xdsl.traits import SymbolTable
+from xdsl.utils.dialect_stub import DialectStub
+
 from . import _version
 
 __version__ = _version.get_versions()["version"]
+
+import importlib.abc
+import importlib.util
+import os
+import sys
+
+
+class CustomFileLoader(importlib.abc.Loader):
```

Comment:
Can you add a bit of documentation here and in the next class?

Reply:
And can we rename the class? :)

Reply:
What a direct attack against state-of-the-art ChatGPT :open_mouth: 
Forgot to rename the boilerplate I kept ahah, will rename more IRDLy